ethcore/trace/src/types/trace.rs

```diff
@@ -33,6 +33,56 @@ pub struct CallResult {
 	pub output: Bytes,
 }
 
+/// `Call` type
```

Comment:
I think we can do better here. Please document what the `CallType` is and what it is used for.